### PR TITLE
Add enriched wallet info endpoint

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Add enriched wallet info endpoint with owners per safe and related booleans addresed to frontend.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: operate/wallet/master.py [hunk 1]
```diff
@@ -393,6 +399,31 @@ def update_backup_owner(
...
+        """Get JSON representation with extended information (e.g., safe owners)."""
+        rpc = None
+        wallet_json = self.json
+
+        if not self.safes:
+            return wallet_json
+
+        owner_sets = set()
+        for chain, safe in self.safes.items():
+            ledger_api = self.ledger_api(chain=chain, rpc=rpc)
```
Comment (jmoreira-valory) [inline operate/wallet/master.py:413]: Remove rpc? Can't be passed to a `@property`.

**Comment by truemiller** [comment]
nice!

**Review by truemiller — APPROVED** [review]

File: operate/wallet/master.py [hunk 2]
```diff
@@ -393,6 +399,32 @@ def update_backup_owner(
...
+            owners = get_owners(ledger_api=ledger_api, safe=safe)
+            owners.remove(self.address)
+            wallet_json["safes"][chain.value] = {
+                wallet_json["safes"][chain.value]: {"owners": owners}
+            }
+            owner_sets.add(frozenset(owners))
+
+        wallet_json["all_chains_same_safe_address"] = len(set(self.safes.values())) == 1
+        wallet_json["all_safe_backup_owners_match"] = len(owner_sets) == 1
+        wallet_json["single_backup_owner_per_safe"] = all(
```
Comment (jmoreira-valory) [inline operate/wallet/master.py:423]: maybe use instead:
`consistent_safe_address`  (= all chains same safe address)
`consistent_safe_owners` (= all chains safes = same set of owners  AND only 1 backup owner)
?

File: operate/wallet/master.py [hunk 3]
```diff
@@ -145,6 +145,12 @@ def update_backup_owner(
         """Update backup owner."""
         raise NotImplementedError()
 
+    # TODO move to resource.py ?
```
Comment (jmoreira-valory) [inline operate/wallet/master.py:148]: @OjusWiZard

**Pull request merged**